sympy/integrals/tests/test_integrals.py

```diff
@@ -2130,3 +2130,14 @@ def test_issue_25886():
             - 1.13875255748434*exp(0.937098661*I))
     F = integrate(f, (x, y, 1.0))
     assert F.is_same(F_exp, math.isclose)
+
+def test_old_issues():
+    #issue 5212
```

Comment:
There should be two blank lines between top-level functions like in the rest of the file.

Also it is better to use an issue URL rather than just a number so that it is clickable and also so that it is clear that it refers to a GitHub issue number.

Reply:
I've done the mentioned changes. Thank you for reviewing.